.github/workflows/build.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           libncurses-dev
           pkg-config
           zlib1g-dev
+          lzma-dev
```

Comment:
we could consider using the nix shell here, actually

Reply:
The disadvantage is if it breaks, you are likely the only one who could fix it

Reply:
you can still go back to doing it "normally"